themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss

```diff
@@ -0,0 +1 @@
+<input $getAttributesHTML('title', 'data-toggle') />
```

Comment:
I'm not even sure what this does - was this for enabling the tooltip on the checkbox?  We don't want to do this?  Can we just delete this file entirely?

Reply:
It's for doing the opposite. Those attributes are "exclusions", not "inclusions".

Reply:
OK sweet